app/Http/Controllers/Admin/StatisticsController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Pterodactyl\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use Pterodactyl\Http\Controllers\Controller;
+use Pterodactyl\Models\Allocation;
+use Pterodactyl\Models\Database;
+use Pterodactyl\Models\Egg;
+use Pterodactyl\Models\Node;
+use Pterodactyl\Models\Server;
+use Pterodactyl\Models\User;
+use JavaScript;
+use Illuminate\Support\Facades\DB;
+use Pterodactyl\Services\DaemonKeys\DaemonKeyProviderService;
```

Comment:
IIRC this should be alphabetized

Reply:
Should be sorted by length, and then alphabetically if anything is the same length. The .phpcs file should handle all of that if you can setup a linter locally.